Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/API/wmb.yaml

```
@@ -94,3 +103,20 @@ apis:
+        tripTransactionId: Text
+      request:
+        type: RequestDetails
+      response:
+        type: DriverReqResp
+  - POST:
+      endpoint: /ui/wmb/requests/{driverRequestId}/cancel
+      auth: TokenAuth PROVIDER_TYPE
+      params:
+        driverRequestId: Text
+      response:
+        type: APISuccess
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)